roles/ovirt-provider-ovn-driver/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- block:
+  - name: Install ovirt-provider-ovn-driver
+    yum:
+        name: ovirt-provider-ovn-driver
+        state: latest
+  - name: Ensure ovn-controller is started
+    service:
+        name: ovn-controller
+        state: started
+        enabled: yes
+  - name: Ensure Open vSwitch is started
+    service:
+        name: openvswitch
+        state: started
+        enabled: yes
+  - name: Configure OVN for oVirt
+    command: >
+        vdsm-tool ovn-config {{ ovn_central }} {{ ovn_tunneling_network }}
+  when:
+    ovn_central | ipaddr
```

Comment:
just a note that you can use:

```yaml
when:
  - ovn_central | ipaddr
  - host_deploy_cluster_version | version_compare('4.2', '>=')
```
When you use list, it's considered as `and`

Reply:
Thanks, done.